### PR TITLE
Barrows and Spooky Forest Hills

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

**Review by Shnupbups — APPROVED** [review]
Looks good to me!

File: src/main/java/com/fabriccommunity/spookytime/world/SpookyChunkGenerator.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
 		
 	});
 	
+	private static final double noiseFrequency = 200;
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/world/SpookyChunkGenerator.java:31]: Static and final field, so `NOISE_FREQUENCY`

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by Shrimpstress — APPROVED** [review]
Looks good to me. Just fix the other requested changes and I’ll be happy to merge this.

**Comment by valoeghese** [comment]
Since I won’t be able to work on modding until Thursday, and the only change requested is a style change, can I merge this?

**Pull request merged**